src/Bicep.LangServer/Deploy/DeploymentCollectionProvider.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Bicep.Core;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public class DeploymentCollectionProvider : IDeploymentCollectionProvider
+    {
+        public DeploymentCollection? GetDeploymentCollection(ArmClient armClient, ResourceIdentifier resourceIdentifier, string scope)
```

Comment:
Is it actually possible for this function to return null?

Reply:
Nope. Updated here:
https://github.com/Azure/bicep/commit/67ff4dea5d71e3f1f1736f79cc89290e85857133

Thanks!